src/parser/jobs/ast/modules/Sect.tsx

```diff
@@ -0,0 +1,253 @@
+import {Trans} from '@lingui/react'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {BuffEvent, CastEvent, Event} from 'fflogs'
+import _ from 'lodash'
+import Module, {dependency} from 'parser/core/Module'
+import Combatants from 'parser/core/modules/Combatants'
+import {Statistics} from 'parser/core/modules/Statistics'
+import Suggestions, {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import SectStatistic from './SectStatistic'
+
+const ASPECTED_ACTIONS = [
+	ACTIONS.ASPECTED_BENEFIC.id,
+	ACTIONS.ASPECTED_HELIOS.id,
+	ACTIONS.CELESTIAL_INTERSECTION.id,
+	ACTIONS.CELESTIAL_OPPOSITION.id,
+]
+
+const DIURNAL_SECT_STATUSES = [
+	STATUSES.ASPECTED_BENEFIC.id,
+	STATUSES.ASPECTED_HELIOS.id,
+	STATUSES.DIURNAL_OPPOSITION.id,
+	STATUSES.NOCTURNAL_INTERSECTION.id,
+	STATUSES.NOCTURNAL_BALANCE.id,
+]
+
+const NOCTURNAL_SECT_STATUSES = [
+	STATUSES.NOCTURNAL_FIELD.id,
+	STATUSES.NOCTURNAL_OPPOSITION.id,
+	STATUSES.DIURNAL_INTERSECTION.id,
+	STATUSES.DIURNAL_BALANCE.id,
+]
+
+const ACTION_STATUS_MAP = {
+	[ACTIONS.ASPECTED_BENEFIC.id]: [
+		STATUSES.ASPECTED_BENEFIC.id,
+		STATUSES.NOCTURNAL_FIELD.id,
+	],
+	[ACTIONS.ASPECTED_HELIOS.id]: [
+		STATUSES.ASPECTED_HELIOS.id,
+		STATUSES.NOCTURNAL_FIELD.id,
+	],
+	[ACTIONS.CELESTIAL_INTERSECTION.id]: [
+		STATUSES.DIURNAL_INTERSECTION.id,
+		STATUSES.NOCTURNAL_INTERSECTION.id,
+	],
+	[ACTIONS.CELESTIAL_OPPOSITION.id]: [
+		STATUSES.DIURNAL_OPPOSITION.id,
+		STATUSES.NOCTURNAL_OPPOSITION.id,
+	],
+}
+
+const SECT_ACTIONS = [
+	ACTIONS.DIURNAL_SECT.id,
+	ACTIONS.NOCTURNAL_SECT.id,
+]
+
+const SECT_STATUSES = [
+	STATUSES.DIURNAL_SECT.id,
+	STATUSES.NOCTURNAL_SECT.id,
+]
+
+const DIURNAL_SECT_BUFF_ABILITY = {
+	name: STATUSES.DIURNAL_SECT.name,
+	guid: STATUSES.DIURNAL_SECT.id,
+	type: 1,
+	abilityIcon: _.replace(_.replace(STATUSES.DIURNAL_SECT.icon, 'https://xivapi.com/i/', ''), '/', '-'),
+}
+
+const NOCTURNAL_SECT_BUFF_ABILITY = {
+	name: STATUSES.NOCTURNAL_SECT.name,
+	guid: STATUSES.NOCTURNAL_SECT.id,
+	type: 1,
+	abilityIcon: _.replace(_.replace(STATUSES.NOCTURNAL_SECT.icon, 'https://xivapi.com/i/', ''), '/', '-'),
+}
+
+// Just in case they go through "THIS MUCH" of the fight with no sect, we're NOT going to babysit the whole log
+// tslint:disable-next-line: no-magic-numbers
```

Comment:
is that seriously throwing a magic number complaint? shouldn't be, if it is lemme know and i'll make sure to check it when i get around to fixing The Linter Situation.

Reply:
It wasn't.. not sure why I was paranoid enough to add that.